package.json

```diff
@@ -75,30 +60,26 @@
     "jsdoc": "3.4.0",
     "jshint": "2.9.1",
     "json-loader": "0.5.4",
-    "load-grunt-config": "0.17.2",
-    "load-grunt-tasks": "3.2.0",
     "memdown": "1.0.0",
+    "mocha": "2.4.5",
```

Comment:
right right, just letting you know.  `nodeapi` i is our _only_ mocha consuming repo, and it's going to momentarily start running the version it its own package (vs. a global executable).  consequently, that ansible bit will be obsolete